### DISCUSSION TITLE
Registration for non-Chinese teams

### DISCUSSION THREAD

**Discussion by thaihaquang** [PaddlePaddle/PaddleSpatial#26] [BaiduKDDCup2022] [answered]
Dear organizers,
I'm sorry if I posted in the wrong topic. I am investigating your competition and want to join it. However, it requires inputting the mobile phone number (i think it should be a Chinese phone number because I input my phone number but did not receive anything) on Personal Social Information page. So Can I join the competition without verification using cell phone?
Thank you.

**Comment by zhoujingbo** [comment, marked as answer]
Hi, 

Our system can be registered with a foreign mobile number. You should input the country calling code,  like +237 with your mobile phone. If you still cannot receive the code, you can send your phone number to our email kddcup2022@baidu.com, we will check it.

Regards

**Reply by thaihaquang** [reply]
Thank you for your support. I used a foreign phone number but have not received the code yet. I will contact the email to check.
